### DISCUSSION TITLE
The kernel keeps restarting when trying to render a scene with propagation paths

### DISCUSSION THREAD

**Discussion by Sunday-AMATARE** [NVlabs/sionna#561] [General]
Hi everyone,
I am new to Sionna and I am working on a project that requires me to render a scene with propagation paths. The rendering seemed to be working just fine before, but now that I am using the same code, it shows "The kernel for ...... appears to have died. It will restart automatically."

Code:
```
paths = scene.compute_paths(max_depth=5,
                                                   num_samples=512)

if colab_compat:
   scene.render("my_cam", paths=paths, show_devices=True, show_paths=True, resolution=resolution);
   raise Exitcell

scene.preview(paths, show_devices=True, show_paths=True)
```

**Comment by faycalaa** [comment]
Hi @Sunday-AMATARE,

Please consider sharing a complete minimal code snippet that reproduces the problem. This will help us assist you more effectively.

**Reply by merlinND** [reply]
Furthermore, please let us know which version of Sionna and Python are you using, whether you are using a GPU or not (and if so, which GPU).
Please also report everything that is logged to the console where you started the Jupyterlab server (assuming that's what you are using) when you execute the cell that crashes.

**Comment by glopesr** [comment]
Did you solve the problem? I am facing the same problem when trying to load a small scene only to test the using sionna RT within a specif area.
